### PR TITLE
Feature/default map

### PR DESCRIPTION
**Pull request by alegnaaived**

### CODE REVIEW

File: src/api/RestAPI.js [hunk 1]
```diff
@@ -295,15 +295,10 @@ class RestAPI {
...
    *
-   * @return {Object} Including Promise with layer object to load in map and source reference to
-   * cancel the request
+   * @return {Promise<Object>} layer object to be loaded in the map
    */
   static requestGeofenceGeometryByArea(areaId, geofenceId) {
-    const source = CancelToken.source();
-    return {
-      request: RestAPI.makeGetRequest(`${areaId}/layers/${geofenceId}`, { cancelToken: source.token }),
-      source,
```
Comment (erikasv) [inline src/api/RestAPI.js:305]: Why did you remove this? I actually think every geometry request should return the cancel token. Once the loading modal is attached only to the map, we'll need to cancel requests

Comment (alegnaaived) [reply src/api/RestAPI.js:305]: Okay, I will keep this to cancel the request

File: src/api/RestAPI.js [hunk 2]
```diff
@@ -342,15 +353,28 @@ class RestAPI {
...
       case 'wetland':
-        return RestAPI.makeGetRequest(`${areaType}/${areaId}/se/layers/Humedal`);
+        return {
+          request: RestAPI.makeGetRequest(`${areaType}/${areaId}/se/layers/Humedal`, { cancelToken: source.token }),
+          source,
+        };
       default:
-        return undefined;
+        return {
+          request: undefined,
```
Comment (erikasv) [inline src/api/RestAPI.js:375]: Please change this for `Promise.reject(new Error('undefined option))` to avoid problems in switchLayer function

**Review by erikasv — CHANGES_REQUESTED** [review]

File: src/api/RestAPI.js (hunk 2 above)
Comment (alegnaaived) [reply src/api/RestAPI.js]: Done

File: src/api/RestAPI.js (hunk 1 above)
Comment (alegnaaived) [reply src/api/RestAPI.js]: Done (with your help) (y)

**Review by erikasv — APPROVED** [review]
